bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/tabs/AjaxBootstrapTabbedPanel.java

```diff
@@ -44,8 +45,9 @@ public void onClick(final Optional<AjaxRequestTarget> targetOptional) {
 				targetOptional.ifPresent(target -> target.add(AjaxBootstrapTabbedPanel.this));
 				onAjaxUpdate(targetOptional);
 			}
-
 		};
+		link.add(Behavior.onAttribute("class", old -> old + (index == getSelectedTab() ? " " + getSelectedTabCssClass() : "")));
```

Comment:
I think I've fixed this with https://github.com/l0rdn1kk0n/wicket-bootstrap/commit/c95661ed224909f28342e45ec11f1c71c5b19241
Is it wrong/broken ?

Reply:
AjaxBootstrapTabbedPanel overrides method from BootstrapTabbedPanel without call to super

I can try to change my code to reuse your solution
(not sure temporary behavior will work)

Reply:
@martin-g ping :)